src/shared/lib/boxPlotUtils.ts

```diff
@@ -0,0 +1,113 @@
+import jStat from 'jStat'
```

Comment:
You will need to delete this file as #1186 is merged

Reply:
Ersin, i can't seem to remember exactly what this was about.  I sent you this file, you committed it and merged it so I just have to accept yours, right?

Reply:
@alisman yes